simulator-windows/CMakeLists.txt

```diff
@@ -2,70 +2,41 @@ cmake_minimum_required(VERSION 3.5)
 project(simulator-windows)
 
 find_package(SystemCLanguage CONFIG REQUIRED)
-#find_package(CUDA REQUIRED)
-#find_package(OpenMP REQUIRED)
-#if(OpenMP_FOUND)
-#message("OPENMP FOUND")
-#endif()
+
+option(USE_CUDA "Default option USE_CUDA is off" off)
+message(STATUS "computation use cuda switch: " ${USE_CUDA})
+if(${USE_CUDA} STREQUAL "on")
+	find_package(CUDA REQUIRED)
+endif(${USE_CUDA} STREQUAL "on")
+
+find_package(OpenMP REQUIRED)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}  -std=c++11 -O3")
-#set(CUDA_NVCC_FLAGS ${CUDA_NVCC_FLAGS})
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Xclang -fopenmp -I/usr/local/opt/libomp/include -L/usr/local/opt/libomp/lib -lomp ")
```

Comment:
The absolute path may cause compiling failure in other computers. You can use find_pageage(OpenMP REQUIRED) to find the path and lib. For more information, please see https://cmake.org/cmake/help/v3.9/module/FindOpenMP.html

Reply:
I have commented this line because this is for macbook compliation. The newest commit has commented it.